cantools/subparsers/dump.py

```diff
@@ -1,3 +1,5 @@
+import curses
```

Comment:
this is a problematic module for some users, see #280.

Reply:
I moved that so that it's lazily imported and falls back to a console width of 80 in case it's missing. See [e61bf69](https://github.com/eerimoq/cantools/pull/283/commits/e61bf699b0a7d9f6d9ab5287dbaa86f71d81e968)